src/steps/10-FinancialDetails/CurrentlyHasFunding.vue

```diff
@@ -1,30 +1,107 @@
 <template>
-  <v-container fluid class="container-max-width">
-    <v-row>
-      <v-col class="col-12">
-        <h1 class="page-header mb-3">
-          Do you currently have funding for your acquisition package?
-        </h1>
-        <div class="copy-max-width">
-          <p class="mb-10">
-           
-          </p>
-        </div>
-       
-      </v-col>
-    </v-row>
-  </v-container>
+  <v-form ref="form" lazy-validation>
+    <v-container class="container-max-width" fluid>
+      <v-row>
+        <v-col class="col-12">
+          <h1 class="page-header mb-3">
+            Do you currently have funding for your acquisition package?
+          </h1>
+          <div class="copy-max-width">
+            <p id="IntroP" class="mb-7">
+              Your Contracting Office may require an interagency agreement (e.g., Fiscal Service
+              Form 7600A/General Terms & Conditions (GT&C) agreement) and/or a funding request
+              (e.g., Fiscal Service Form 7600B or Military Interdepartmental Purchase Request
+              (MIPR)) to procure cloud service offerings from JWCC on your agency’s behalf.
+            </p>
+            <ATATRadioGroup
+                :card="true"
+                :items="radioButtonItems"
+                :rules="[$validators.required('Please select an option.')]"
+                :value.sync="selectedHasFunding"
+                class="max-width-640 mb-5"
+            />
+          </div>
+        </v-col>
+      </v-row>
+    </v-container>
+  </v-form>
 </template>
+
 <script lang="ts">
+/* eslint-disable camelcase */
 import { Component, Mixins } from "vue-property-decorator";
 
-
+import ATATRadioGroup from "@/components/ATATRadioGroup.vue";
+import { RadioButton } from "../../../types/Global";
+import { hasChanges } from "@/helpers";
+import FinancialDetails from "@/store/financialDetails";
 import SaveOnLeave from "@/mixins/saveOnLeave";
+import {isDitcoUser} from "@/store/acquisitionPackage";
+import {routeNames} from "@/router/stepper";
+
+@Component({
+  components: {
+    ATATRadioGroup
+  }
+})
 
-@Component({})
 export default class CurrentlyHasFunding extends Mixins(SaveOnLeave) {
+  private selectedHasFunding = "";
+  private radioButtonItems: RadioButton[] = [
+    {
+      id: "HasFundingCheckbox",
+      label: "Yes. I have a GT&C, as well as a FS Form 7600B or MIPR.",
+      value: "HAS_FUNDING"
+    },
+    {
+      id: "NoFundingCheckbox",
+      label:
+          "No. I need to obtain funding documents.<br/>" +
+          "<span style='color: #41494E'>We’ll generate a draft of your package to " +
+          "submit to your financial office next.</span>",
+      value: "NO_FUNDING",
```

Comment:
remove the span in line 60, so the error class will decorate all text...
![image](https://github.com/dod-ccpo/atat-web-ui/assets/84856468/188397ab-7da3-49ec-9fb9-f0bbdbec4acd)

Reply:
Removed the span and verified that all text now turns red upon validation failure